java/vector/pom.xml

```diff
@@ -133,6 +133,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.1</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <artifactSet>
+                <excludes>
+                  <exclude>com.google.flatbuffers:*</exclude>
+                  <exclude>io.netty:*</exclude>
```

Comment:
are these necessary for this particular issue?

Reply:
Yes, seems arrow-vector maven-shade-plugin changes the transitive dependencies, if we do not exclude these, build fail in arrow-tool like:
`[WARNING] Unused declared dependencies found:`
`[WARNING]    org.slf4j:slf4j-api:jar:1.7.25:compile`
`[WARNING]    com.fasterxml.jackson.core:jackson-databind:jar:2.9.8:compile`
`[WARNING]    com.fasterxml.jackson.core:jackson-core:jar:2.9.8:compile`

if we just remove above direct dependency from arrow-tool, build still fail:
`[WARNING] Used undeclared dependencies found:
[WARNING]    org.slf4j:slf4j-api:jar:1.7.25:compile
`
Similar problems in other modules like arrow-adapter.

Seems two way to solve this problem:
1、add excludes in maven-shade-plugin
2、remove these direct dependency from tools、adapter and add some exclusions in slf4j in order to solve error mentioned above.

Reply:
Is some action needed on this before this can be merged?

Reply:
Since this could pass build and has no effect on other module, i think no more actions are needed, thanks